xmtp_mls/migrations/2024-12-04-194513_hmac-keys-records-table/up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE "hmac_key_records"(
+    -- Group ID that the Hmac keys are associated with
+    "group_id" BLOB NOT NULL,
```

Comment:
I did this based off of the hmac keys work in V2 but @codabrink mentioned we might be doing this based on inboxId now. So maybe this is irrelevant in V3?